gecko/core/engine/buffer.py

```
@@ -128,24 +196,33 @@ def _clean_arguments(self, raw_json: str) -> str:
             return cleaned
-        except Exception:
-            # 如果还无法解析，记录警告并返回原始内容，让上层 Engine/ToolBox 报错
-            # 这样可以在 Tool Output 中反馈给 LLM，让它自己修正
-            logger.warning(f"Failed to clean JSON arguments: {raw_json[:50]}...")
-            return raw_json
\ No newline at end of file
+        except json.JSONDecodeError as e:
+            # [P1-4 Fix] 如果还无法解析，返回空对象而非脏数据
+            # 这避免工具执行时因脏 JSON 而崩溃
+            logger.warning(
+                f"Failed to clean JSON arguments, returning empty dict",
```

Ruff (0.14.7)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)